docs/Installation-Windows.md

```diff
@@ -152,7 +161,7 @@ GPU). This is a guide for advanced users who want to train using GPUs.
 Additionally, you will need to check if your GPU is CUDA compatible. Please
 check Nvidia's page [here](https://developer.nvidia.com/cuda-gpus).
 
-As of the ML-Agents toolkit v0.4, only CUDA v9.0 and cuDNN v7.0.5 is supported.
+As of the ML-Agents toolkit v0.6, only CUDA v9.0 and cuDNN v7.0.5 is supported.
```

Comment:
similar to above - remove the v0.6 as we'll always need to uptick

Reply:
Fixed both places.